src/main/java/com/cloudant/client/org/lightcouch/NoResourceException.java

```diff
@@ -1,30 +0,0 @@
-/**
```

Comment:
Why was this removed? Presumably this basically does the same job as `NoResourceException` and is therefore redundant?